lib/matplotlib/figure.py

```diff
@@ -417,6 +417,108 @@ def _set_dpi(self, dpi):
         self.callbacks.process('dpi_changed', self)
     dpi = property(_get_dpi, _set_dpi)
 
+    def set_axis_break_y(self, bottom, top, *args):
+        """
+        Add a line break on the y-axis
+
+        Parameters
+        ----------
+
+        bottom, top : scalar
+            Controls where the line break is on the y-axis
+
+        args*:
+            A variable length argument, allowing for multiple *x*, *y* pairs
+
+        Returns
+        -------
+
+        break : tuple
+            The break values (bottom, top)
+
+        """
+
+        if (bottom >= top):
+            raise ValueError
+        (ax, ax2) = self.subplots(2, 1, sharex=True)
```

Comment:
Looks like this should rather be a method on the Axes class, so that you can "break" any existing axes, rather than being limited to a single broken axes per figure.